packages/babel-plugin/src/index.js

```diff
@@ -20,138 +20,194 @@ const properties = [
 
 const LOADABLE_COMMENT = '#__LOADABLE__'
 
-const loadablePlugin = declare((api, { defaultImportSpecifier = 'loadable' }) => {
-  const { types: t } = api
-
-  function collectImportCallPaths(startPath) {
-    const imports = []
-    startPath.traverse({
-      Import(importPath) {
-        imports.push(importPath.parentPath)
-      },
-    })
-    return imports
-  }
+/**
```

Comment:
The golden rule of development is to __never__ refactor and change simultaneously.
Look like this file is not _changed_, however from the git point of view - everything changed.

I also cannot spot any change, but that does not mean there isn't any.

While I do appreciate typings and docs
- [ ] please remove unrelated changes

Reply:
Cool, thanks for the tip! Done :rocket: